frontends/main/src/app-pages/DashboardPage/OrganizationContent.test.tsx

```diff
@@ -170,11 +168,11 @@ describe("OrganizationContent", () => {
 
       // Check based on the actual enrollment status, not array position
       if (course.enrollment?.status === EnrollmentStatus.Enrolled) {
-        expect(indicator).toHaveTextContent("Enrolled")
+        expect(indicator).toHaveTextContent(/^Enrolled$/)
```

Comment:
As originally written, a bunch of tests in this file passed irrespective of whether the text was "Enrolled" or "Not Enrolled". `toHaveTextContent` is only an exact match if you use regex to specify start/stop. If you pass a string, it just checks for inclusion.